hack/go-linter.sh

```diff
@@ -9,6 +9,7 @@ header_text "Running golangci-lint"
 golangci-lint run --disable-all \
     --deadline 5m \
     --enable=nakedret \
+    --enable=ineffassign \
```

Comment:
```suggestion
    --enable=ineffassign
```

Reply:
I would like to keep with `\` because of the merges.